docs/admin-guide/install-buildout.md

```diff
@@ -75,21 +77,28 @@ eggs =
     Plone
 ```
 
+Install Buildout script in bin directory.
+
+```shell
+./venv/bin/buildout bootstrap
```

Comment:
This isn't needed. Buildout was already installed as part of the `pip install` command on line 56.

Reply:
it does not install buildout, but it creates the bin/buildout script, so that you don't need to prefix the buildout run with the venv folder.all the time.